examples/cloudml-churn-prediction/trainer/trainer/model.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TensorFlow model definition."""
+
+import tensorflow as tf
+
+from trainer import metrics
+
+
+def get_feature_columns(tf_transform_output, exclude_columns=None):
+    """Returns list of feature columns for a TensorFlow estimator.
+
+    Args:
+        tf_transform_output: tensorflow_transform.TFTransformOutput.
+        exclude_columns: `tf_transform_output` column names to be excluded
+            from feature columns.
+
+    Returns:
+        List of TensorFlow feature columns.
+    """
+
+    tf_numeric_types = [
+        tf.float16,
+        tf.float32,
+        tf.float64,
+        tf.int8,
+        tf.int16,
+        tf.int32,
+        tf.int64,
+    ]
+
+    feature_columns = []
+    feature_spec = tf_transform_output.transformed_feature_spec()
+
+    if exclude_columns:
+        for col in exclude_columns:
```

Comment:
Can refactor to avoid so much nesting. 
```python
    if exclude_columns: 
        feature_spec = {col: val for (col, val) in feature_spec if col not in exclude_columns}
```